### DISCUSSION TITLE
Is there a way to limit fan speed in this scenario?

### DISCUSSION THREAD

**Discussion by TempleKa** [Rem0o/FanControl.Releases#2352] [General]
![fancontrol](https://github.com/Rem0o/FanControl.Releases/assets/78983444/2bc3afa6-7e2b-4dee-8138-b81ee596db28)

This above is my fan control setup. My CPU AIO fans and GPU fans are on their curves and the case fans work in a mix, spinning faster or slower depending on CPU or GPU fan speed.

The thing is I want my CPU and GPU fans to spin all the way to 100% but I don't want the rest of the case fans to do that. I would like the case fans to spin in sync with either the CPU or GPU until they reach a limit of say 70-80%. After that threshold is passed the CPU and GPU fans would continue to ramp but not the case fans.

So in my case I would like to limit **FRONT Intake Fans**, **BOTTOM Intake Fans** and **REAR Exhaust Fan** to 70-80% but the rest with no limit.

Is this possible in any way?

**Comment by Rem0o** [comment]
Create 2 new separate curves for your CPU and GPU cooler with those characteristics, and leave your existing curves as is.

**Reply by TempleKa** [reply]
So I think I figured it out. I created 2 new graph curves for CPU and GPU that go to a maximum of 60%. I than edited the Mix Curve in my initial picture by adding these 2 new curves and deleted the old ones.

So this is how it looks now on full load of the CPU (there's an offset of 30% on Bottom intake). Hopefully it's correct.

![new](https://github.com/Rem0o/FanControl.Releases/assets/78983444/fe1fc0b0-913a-40b5-b536-381b0ee1c41e)

**Reply by Rem0o** [reply]
The point was not to delete the old ones. You want different behavior for your case fans and CPU cooler, so create different curves for them. You should leave your original mix alone.

**Reply by TempleKa** [reply]
Now I'm confused. Didn't I just do that? I mean the CPU AIO and GPU fans are able to spin up to 100% according to their respective curves (CPU Temp and GPU Temp). The other case fans spin according to the 2 new curves (CPU and GPU) in a Mix but only up to 60%.

I tested it by putting load on the CPU and GPU separately and it seems to work. All case fans don't pass the 60% mark while the AIO and GPU fans go up to 100% if needed depending on their temperature. That's what I needed, but maybe I wasn't explaining properly or I was misunderstood.

**Reply by Rem0o** [reply]
We may got confused, but it seems like you got it figured out 👌
